libafl/src/corpus/mod.rs

```diff
@@ -86,8 +89,7 @@ macro_rules! random_corpus_id_with_disabled {
 
 /// Corpus with all current [`Testcase`]s, or solutions
 pub trait Corpus: Sized {
-    /// The type of input contained in this corpus
-    type Input;
+    type Input: Debug;
```

Comment:
Shouldn't be here. It makes only a couple of the constraints later simpler, but IMO makes things less clean.

Is there a specific reason you added this?